src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -300,8 +303,24 @@
 
     public static final Key<Value<Boolean>> ELDER_GUARDIAN = KeyFactory.fake("ELDER_GUARDIAN");
 
+    /**
+     * Represents the {@link Key} for representing if the exact teleport location
```

Comment:
'represent' used twice in one sentence reads a bit oddly. Don't need the first one. could just read "Key for representing..."